website/static/index.html

```diff
@@ -228,22 +212,29 @@ <h4 class="accordion-heading"><span><strong class="faq-q">What are the future pl
             <div class="icon-2 accordion-icon">keyboard_arrow_down</div>
           </div>
           <div class="accordion-item-content">
-            <p class="faq-a">Current contributors across Onehouse, Microsoft, Google, and others are planning to incubate OneTable into the Apache Software Foundation. Stay tuned for more updates.</p>
+            <p class="faq-a">Current contributors across Onehouse, Microsoft, Google, and others are helping to incubate Apache XTable™ into the Apache Software Foundation. Stay tuned for more updates.</p>
           </div>
         </div>
       </div>
     </div>
   </div>
   <section class="footer-subscribe">
     <div class="main-container-footer">
-      <div class="div-block-25"><img src="images/OneTable-F.png" loading="lazy" alt="" class="image-8"></div>
+      <div class="div-block-25"><img src="images/xtable-white.png" loading="lazy" alt="" class="image-8"></div>
       <div class="footer-block-three">
         <a href="https://onetable.dev/docs/setup/" class="footer-link-three">Docs</a>
-        <a href="https://github.com/onetable-io/onetable" class="footer-link-three">GitHub</a>
-        <a href="https://www.linkedin.com/company/onetable-oss/" class="footer-link-three">LinkedIn</a>
-        <a href="https://twitter.com/OnetableOSS" class="footer-link-three l">Twitter</a>
+        <a href="https://github.com/apache/incubator-xtable" class="footer-link-three">GitHub</a>
+        <a href="https://www.linkedin.com/company/apache-xtable/" class="footer-link-three">LinkedIn</a>
+        <a href="https://twitter.com/apachextable" class="footer-link-three l">Twitter</a>
       </div>
-      <div class="footer-copyright">                          </div>
+    </div>
+    <div class="footer__bottom text--center"><div class="footer__copyright" style="color: white;text-align: center;">
+      <div style="margin-top: 20px">
+        <a href="https://incubator.apache.org/" target="_blank"><img style="height:40px; margin-bottom: 10px; margin-top: 10px" alt="Apache Software Foundation" src="/images/apache-incubator.svg"></a>
```

Comment:
Wouldn't be better to use hyperlinks for such "external" images instead of committing and maintaining them to the repo?

Some incubator logos can be found below:
* https://svn.apache.org/repos/asf/comdev/project-logos/originals/incubator.svg
* https://incubator.apache.org/guides/press-kit.html

If none of them works then maybe we should contribute the new one to one of those places WDYT?

Reply:
yes I can grab the apache related ones from there, and after vote passes upload the Apache Xtable ones. There will be other external images not in these repos that I think will need to land in the repo. After this initial name change PR on devlist email we discussed we will redesign how the website is maintained in GitHub to follow the Apache way.

Reply:
Ok I got the incubator logo referenced from external, thanks for the tip